api/web3server_utils.go

```diff
@@ -87,11 +87,19 @@ func ethAddrToIoAddr(ethAddr string) (address.Address, error) {
 	return address.FromHex(ethAddr)
 }
 
+func ioAddrToEvmAddr(ioAddr string) (common.Address, error) {
+	address, err := address.FromString(ioAddr)
+	if err != nil {
+		return common.Address{}, output.NewError(output.ConvertError, "", err)
```

Comment:
`output.NewError(output.ConvertError, "", err)` could be replaced by `errInvalidFormat`